TicketBookingAPI/Program.cs

```diff
@@ -11,7 +12,7 @@
 builder.Services.AddSwaggerGen();
 builder.Services.AddDbContext<TicketBookingDbContext>(op =>
     op.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
-
+builder.Services.AddTransient<IUnitOfWork, UnitOfWork>();
```

Comment:
why don't we use .AddScoped?

Reply:
Yes, this is our mistake. AddScoped will give better performance than addTransient, we did not notice this issue and will fix in next commit.

Reply:
should change and complete in one PR